basics/basic-types.md

```diff
@@ -0,0 +1,118 @@
+# Elementare Datentypen
+
+D bietet eine Vielzahl an elementaren Datentypen, die immer die gleiche Größe haben -
+**unabhängig** von der Platform.
+Die einzige Ausnahme ist der Typ `real` für welchen der Compiler
+die höchstmöglichste Genauigkeit für Gleitkommazahlen wählt.
+Es besteht kein Unterschied zwischen der Größe eines Ganzzahltypen - unabhängig davon
+ob die Anwendung für eine 32-bit oder 64-bit Architektur kompiliert wird.
+
+<table class="table table-hover">
+<tr><td width="250px"><code class="prettyprint">bool</code></td> <td>8-bit</td></tr>
+<tr><td><code class="prettyprint">byte, ubyte, char</code></td> <td>8-bit</td></tr>
+<tr><td><code class="prettyprint">short, ushort, wchar</code></td> <td>16-bit</td></tr>
+<tr><td><code class="prettyprint">int, uint, dchar</code></td> <td>32-bit</td></tr>
+<tr><td><code class="prettyprint">long, ulong</code></td> <td>64-bit</td></tr>
+</table>
+
+#### Gleitkommatypen
+
+<table class="table table-hover">
+<tr><td width="250px"><code class="prettyprint">float</code></td> <td>32-bit</td></tr>
+<tr><td><code class="prettyprint">double</code></td> <td>64-bit</td></tr>
+<tr><td><code class="prettyprint">real</code></td> <td>abhängig von der Platform, 80-bit für Intel x86 32-bit</td></tr>
+</table>
+
+Der Präfix `u` kennzeichnet Typen ohne Vorzeichen (vom Englischen `unsigned`).
+Ein `char` ist ein UTF-8 Zeichen, `wchar` ein UTF-16 Zeichen and `dchar`
```

Comment:
Is this accurate though? A char is only 1 byte and represents a byte in a UTF-8 string, you might need multiple chars to represent 1 UTF-8 character. You have dchar for full characters without needing to have multiple of them.

This code for example works different than you might expect:
```d
import std.stdio;
void main() {
	string s = "Ω";
	writefln("%s (%s)", s[0], cast(int) s[0]);
}
```

Output: `� (206)`